futures-core/src/task/mod.rs

```diff
@@ -1,31 +1,7 @@
-//! Tasks used to drive a future computation
+//! Task notification.
```

Comment:
This whole file is awesome! I'm so excited to be able to point people to these docs :heart: